pkgs/development/libraries/libdisplay-info/default.nix

```diff
@@ -0,0 +1,41 @@
+{ lib
+, stdenv
+, fetchFromGitLab
+, meson
+, pkg-config
+, ninja
+, python3
+, hwdata
+, edid-decode
+}:
+
+stdenv.mkDerivation rec {
+  pname = "libdisplay-info";
+  version = "0.1.0";
+
+  src = fetchFromGitLab {
+    domain = "gitlab.freedesktop.org";
+    owner = "emersion";
+    repo = pname;
+    rev = version;
+    sha256 = "sha256-jfi7RpEtyQicW0WWhrQg28Fta60YWxTbpbmPHmXxDhw=";
+  };
+
+  nativeBuildInputs = [ meson pkg-config ninja edid-decode python3 ];
+
+  buildInputs = [ hwdata ];
+
+  prePatch = ''
```

Comment:
Patch phase comes after unpack phase and before configure phase.
In this case, please, move `prePatch` to after `src` before `nativeBuildInputs`.

Reply:
If some coding-style rule is not in [the manual](https://nixos.org/manual/nixpkgs/unstable/#id-1.6) nor in the [CONTRIBUTING docs](https://github.com/NixOS/nixpkgs/blob/3d53723112c6045bd6534ebeb536056fec5a16f8/CONTRIBUTING.md#submitting-changes), then I don't see reasons to comply.

The biggest reason for me to leave ArchLinux was that the TUs would come up with rules from their mind and argue from whatever favors their taste.

Considering, we have great tools like `nixpkgs-fmt` (that I used in this file). We should just enforce its usage through CI and put all the tasteful rules in there.

Everything that is not documented, open space for conflicting argues and takes contributors and maintainers time in a boomerang during code-review that should focus on code instead of style.

Reply:
I don't believe this case specifically blocks code readability.